pom.xml

```diff
@@ -173,6 +191,11 @@
       <id>ImageJ Public</id>
       <url>http://maven.imagej.net/content/repositories/public/</url>
     </repository>
+    <repository>
+      <id>jcenter-snapshots</id>
```

Comment:
Is this needed for something? The new dependencies should all be on Maven Central